web-client/public/js/setup-handlers.js

```diff
@@ -1,4 +1,6 @@
 import { updateApp } from "./update-app";
+import { modifyChargeParameter } from "./update-app";
+import { modifyLinkDistanceParameter } from "./update-app";
```

Comment:
You can group multiple imports from the same file into a single list (actually, just like below).